components/Map/index.jsx

```diff
@@ -111,7 +107,7 @@ class MapContainer extends React.Component {
 
     if (
       prevProps.activeMode !== activeMode ||
-      prevProps.pins !== pins ||
+			prevProps.councilId !== councilId ||
```

Comment:
replace tabs with spaces

Reply:
Moved them all to one line as it was showing the correct format on my editor.